recipes/magics-python/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "magics-python" %}
+{% set pypi_name = "Magics" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: b852ddc368116f9a3a8d9e778e7d82eb2f086601c5ff3cc61e2ed2928812d498
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  imports:
```

It's not ideal but better than nothing:
```suggestion
  commands:
    - test -f $(python -c "import site; print(site.getsitepackages()[0])")/Magics/__init__.py
```